packages/expo-brightness/ios/EXBrightness/EXSystemBrightnessRequester.m

```diff
@@ -0,0 +1,23 @@
+// Copyright 2016-present 650 Industries. All rights reserved.
+
+#import <EXBrightness/EXSystemBrightnessRequester.h>
+
+@implementation EXSystemBrightnessRequester
```

Comment:
I think we should call these requesters as permission requesters, so let's change it to `EXSystemBrightnessPermissionRequester`